tsconfig.json

```diff
@@ -9,5 +9,5 @@
     "strict": true,
     "target": "ES2017"
   },
-  "include": ["src"]
+  "include": ["src", "test"]
```

Comment:
With this setting, could test files end up in the build without us knowing?

What do you think about defining two tsconfigs, one for dev and one for building? See: https://github.com/MetaMask/metamask-module-template/blob/main/tsconfig.build.json#L11

Reply:
Done!